bot/cogs/bot.py

```diff
@@ -75,40 +75,54 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    def codeblock_stripping(self, msg: str):
+        """
+        Strip msg in order to find Python code.
+
+        Tries to strip out Python code out of msg and returns the stripped block or
+        None if the block is a valid Python codeblock.
+        """
+        if msg.content.count("\n") >= 3:
+            # Filtering valid Python codeblocks and exiting if a valid Python codeblock is found
+            if re.search(msg.content, "```(python|py)\n((?:.*\n*)+)```", re.IGNORECASE):
+                return None
+            else:
+                # Stripping backticks from every line of the message.
+                content = ""
+                for line in msg.content.splitlines():
+                    content += line.strip("`") + "\n"
+
+                # Remove "Python" or "Py" from top of the message if exists
+                if content.lower().startswith("python"):
+                    content = content[6:]
+                elif content.lower().startswith("py"):
+                    content = content[2:]
+                # Strip again to remove the whitespace(s) left before the code
+                # If the msg looked like "Python <code>" before removing Python
+                content = content.strip()
+                return content
+
     async def on_message(self, msg: Message):
         if msg.channel.id in self.previous_format_times:
```

Comment:
Can you change this variable name? `previous_format_times` doesn't seem to make sense anymore. format? what format. what did we format?